java-operators/test/com/baeldung/java/operators/UnaryTest.java

```diff
@@ -0,0 +1,73 @@
+package com.baeldung.java.operators;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class UnaryTest {
+	
+	@Test
+	public void whenUseTheOperatorPlusAndMinus() {
+		
+		int number = 3;
+		int numberChangedToNegative = - (number);
+		
+		assertEquals(numberChangedToNegative, -3);
+		
+		char charA = 'A';
+		int intvalueofTheCharA = + charA;
+		
+		assertEquals(intvalueofTheCharA, 65);
+		
+	}
+	
+
+	@Test
+	public void whenUseTheOperatorIncrementAndDecrement() {
+		
+		int number = 3;
+		number--;
+		
+		assertEquals(number, 2);
+		
+		number++;
+		
+		assertEquals(number, 3);
+		
+	}
+	
+	@Test
+	public void whenUseTheOperatorIncrementAndDecrementInTheMoment() {
```

Comment:
Watch out when you use the And word in a test, it can give you a hint that you can split the test into two tests.